core/src/main/java/hudson/PluginManager.java

```diff
@@ -177,6 +178,31 @@
     /** Custom plugin manager system property or context param. */
     public static final String CUSTOM_PLUGIN_MANAGER = PluginManager.class.getName() + ".className";
 
+    private static final Logger LOGGER = Logger.getLogger(PluginManager.class.getName());
+
+    /**
+     * Time elapsed between retries to check the updates sites.
+     */
+    public static int checkUpdateSleepTimeMillis;
+
+    /**
+     * Number of attempts to check the updates sites.
+     */
+    public static int checkUpdateAttempts;
+
+    static {
+        try {
+            // Secure initialization
+            checkUpdateSleepTimeMillis = SystemProperties.getInteger(PluginManager.class.getName() + ".checkUpdateSleepTimeMillis", 1000);
+            checkUpdateAttempts = SystemProperties.getInteger(PluginManager.class.getName() + ".checkUpdateAttempts", 1);
+        } catch(Exception e) {
```

Comment:
As you are passing default value, there is no reason to catch an exception there. The only problem that could arise is a `null` returned by the method but with the default value, there is no risk. That could simplify a bit the code there.